code/modules/reagents/chems/chems_ethanol.dm

```diff
@@ -7,46 +8,35 @@
 	ignition_point = T0C+150
 	accelerant_value = FUEL_VALUE_ACCELERANT
 	solvent_power = MAT_SOLVENT_MODERATE
-	uid = "chem_ethanol"
 
-	boiling_point = T0C + 78.37
+	boiling_point = null // Pure ethanol boils, the rest has to separate first.
 
 	heating_message = "boils away its ethanol content, leaving pure water."
 	heating_point = T0C + 78.37
 	heating_products = list(
-		/decl/material/liquid/ethanol = 0.75,
-		/decl/material/liquid/water =   0.25
+		/decl/material/liquid/ethanol/spirits = 0.75,
+		/decl/material/liquid/water           = 0.25
 	)
-	bypass_heating_products_for_root_type = /decl/material/liquid/ethanol
 
 	chilling_message = "separates as its water content freezes, leaving pure ethanol."
 	chilling_point = T0C
 	chilling_products = list(
-		/decl/material/liquid/ethanol = 0.75,
-		/decl/material/solid/ice =      0.25
+		/decl/material/liquid/ethanol/spirits = 0.75,
+		/decl/material/solid/ice              = 0.25
 	)
-	bypass_chilling_products_for_root_type = /decl/material/liquid/ethanol
 	affect_blood_on_ingest = FALSE // prevents automatic toxins/inebriation as though injected
 	affect_blood_on_inhale = FALSE
-	can_boil_to_gas = TRUE
-	temperature_burn_milestone_material = /decl/material/liquid/ethanol
```

Comment:
this reminded me that because wood is an abstract material now, this doesn't show wood burning anymore i think
```dm
		for(var/decl/material/material as anything in decls_repository.get_decls_of_subtype_unassociated(/decl/material))
			if(material.type != material.temperature_burn_milestone_material)
				continue
```
get_decls_of_subtype_unassociated can never return abstract decls unless they have the abstract decl init flag, which materials don't. so there will never be an instantiated type where `material.type != material.temperature_burn_milestone_material` succeeds for like. wood. this in fact was done explicitly because alcohol was not an abstract reagent and i think switching it back to true/false would be better (although we probably want to deduplicate it somehow anyway to avoid having burning points listed for every single type of wood?)

Reply:
i was wrong, it was moved to oak LMFAO